api/cmd/portainer/main.go

```diff
@@ -171,6 +172,19 @@ func retrieveFirstEndpointFromDatabase(endpointService portainer.EndpointService
 func main() {
 	flags := initCLI()
 
+	if *flags.CheckHealth {
+		statuscode, err := http.HealthCheck("localhost" + *flags.Addr)
+		if err == nil {
+			if statuscode < 400 {
```

Comment:
I believe that any status other than 200 should raise an error.

Reply:
I used `<400`, because that's the implementation in the reference: https://github.com/wtfcoderz/static-healthcheck/blob/master/healthcheck.go

However, I changed it now, to only accept `200`.

Reply:
I used `<400`, because that's the implementation in the reference: https://github.com/wtfcoderz/static-healthcheck/blob/master/healthcheck.go

However, I changed it now, to only accept `200`.

Reply:
I used `<400`, because that's the implementation in the reference: https://github.com/wtfcoderz/static-healthcheck/blob/master/healthcheck.go

However, I changed it now, to only accept `200`.